javalanglib/src/main/scala/java/lang/Integer.scala

```diff
@@ -81,20 +91,40 @@ object Integer {
         }
         val res = js.Dynamic.global.parseInt(s, radix).asInstanceOf[scala.Double]
 
-        if (res.isNaN || res > MAX_VALUE || res < MIN_VALUE)
+        @inline def isOutOfBounds(): scala.Boolean = {
+          if(signed) res > MAX_VALUE || res < MIN_VALUE
```

Comment:
Missing space after `if`.
Also, no need for the `()` in `isOutOfBounds()`, since this method does not have side effects.